mappings/net/minecraft/world/gen/feature/TwistingVinesFeature.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_4953 net/minecraft/world/gen/feature/TwistingVinesFeature
+	METHOD method_25986 isNotSuitable (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;)Z
+		ARG 1 world
+		ARG 2 pos
+	METHOD method_25987 generateColumn (Lnet/minecraft/class_1936;Ljava/util/Random;Lnet/minecraft/class_2338$class_2339;III)V
```

Comment:
```suggestion
	METHOD method_25987 generateVines (Lnet/minecraft/class_1936;Ljava/util/Random;Lnet/minecraft/class_2338$class_2339;III)V
```

Reply:
This is generating a single column of vine. `generateVines` make people unsure if it is generating a group of columns or a single column.

Reply:
```suggestion
	METHOD method_25987 generateVineColumn (Lnet/minecraft/class_1936;Ljava/util/Random;Lnet/minecraft/class_2338$class_2339;III)V
```